03-app-instrumentation.md

```diff
@@ -40,9 +40,67 @@ the logs of the frontend service should contain `trace_id` and `span_id`
 Finally, look into the `index.js` file once again, there are a few additional
 `TODOs` for you!
 
+## Deploy the application
+
+Run the following command to deploy the sample application to your cluster:
+
+```bash
+kubectl -f https://github.com/pavolloffay/kubecon-eu-2023-opentelemetry-kubernetes-tutorial/blob/main/app/k8s.yaml
+```
+
+After a short while, verify that it has been deployed successfully:
+
+```bash
+$ kubectl get all -n tutorial-application
+NAME                                       READY   STATUS    RESTARTS   AGE
+pod/loadgen-deployment-5cc46c7f8c-6wwrm    1/1     Running   0          39m
+pod/backend1-deployment-69bf64db96-nhd98   1/1     Running   0          19m
+pod/frontend-deployment-bdbff495f-wc48h    1/1     Running   0          19m
+pod/backend2-deployment-856b75d696-d4m6d   1/1     Running   0          19m
+
+NAME                       TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)    AGE
+service/backend1-service   ClusterIP   10.43.194.58   <none>        5000/TCP   39m
+service/backend2-service   ClusterIP   10.43.176.21   <none>        5165/TCP   39m
+service/frontend-service   ClusterIP   10.43.82.230   <none>        4000/TCP   39m
+
+NAME                                  READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/loadgen-deployment    1/1     1            1           39m
+deployment.apps/backend1-deployment   1/1     1            1           39m
+deployment.apps/frontend-deployment   1/1     1            1           39m
+deployment.apps/backend2-deployment   1/1     1            1           39m
+```
+
 ## Auto-instrumentation
 
-TODO Instrument other services with auto-instrumentation.
+The OpenTelemetry Operator supports injecting and configuring
```

Comment:
Does it make sense to link to https://opentelemetry.io/docs/reference/specification/sdk-environment-variables/#exporter-selection somewhere?